pkg/gadgettracermanager/k8s/k8s.go

```diff
@@ -59,9 +57,10 @@ func NewK8sClient(nodeName string) (*K8sClient, error) {
 		return nil, fmt.Errorf("failed to get node %w", err)
 	}
 
-	// get a CRI client to talk to the CRI handling pods in this node
-	// TODO: when to close it?
-	criClient, err := newCRIClient(node)
+	// Get a runtime client to talk to the container runtime handling pods in
+	// this node. TODO: when to close it?
```

Comment:
No related to this PR: This TODO hasn't been solved, I think were leaking the connection to the client in https://github.com/kinvolk/inspektor-gadget/blob/a2a95ec9729b0203cb30fb60ff4073aa64b88df1/pkg/container-collection/options.go#L219, there should be a call to `k8sClient.CloseRuntimeClient()` there before returning. The ContainerCollectionsOptions are also leaking as they never call any Close() method, I opened https://github.com/kinvolk/inspektor-gadget/issues/624 to track this down.